extensions/ql-vscode/src/packaging.ts

```diff
@@ -0,0 +1,144 @@
+import { CodeQLCliServer } from './cli';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import {
+  getOnDiskWorkspaceFolders,
+  showAndLogErrorMessage,
+  showAndLogInformationMessage,
+} from './helpers';
+import { window } from 'vscode';
+import { ProgressCallback } from './commandRunner';
+
+const CORE_PACKS = [
+  'codeql/cpp-all',
+  'codeql/csharp-all',
+  'codeql/go-all',
+  'codeql/java-all',
+  'codeql/javascript-all',
+  'codeql/python-all',
+  'codeql/ruby-all',
+];
```

Comment:
Hard-coded these, but maybe there's a way to list the "core" packs programmatically?

Reply:
Could you remind me of the intended audience for the new commands? I think that influences the choice of packs.
- Query writers getting set up to write new queries and query packs without having to clone the `codeql` repo?
- Security teams running the default queries and examining the results? (If so, what does VS Code make easier compared to using the command line or code scanning?)
- Both?